<dec f='codebrowser/Include/pyerrors.h' l='53' type='PyObject *'/>
<offset>640</offset>
<use f='codebrowser/Modules/_io/bufferedio.c' l='800' u='r' c='_PyIO_trap_eintr'/>
<use f='codebrowser/Modules/_io/bufferedio.c' l='801' u='r' c='_PyIO_trap_eintr'/>
<use f='codebrowser/Objects/exceptions.c' l='896' u='w' c='oserror_init'/>
<use f='codebrowser/Objects/exceptions.c' l='1048' u='r' c='OSError_clear'/>
<use f='codebrowser/Objects/exceptions.c' l='1048' u='w' c='OSError_clear'/>
<use f='codebrowser/Objects/exceptions.c' l='1070' u='r' c='OSError_traverse'/>
<use f='codebrowser/Objects/exceptions.c' l='1070' u='r' c='OSError_traverse'/>
<use f='codebrowser/Objects/exceptions.c' l='1108' u='r' c='OSError_str'/>
<use f='codebrowser/Objects/exceptions.c' l='1108' u='r' c='OSError_str'/>
<use f='codebrowser/Objects/exceptions.c' l='1114' u='r' c='OSError_str'/>
<use f='codebrowser/Objects/exceptions.c' l='1114' u='r' c='OSError_str'/>
<use f='codebrowser/Objects/exceptions.c' l='1119' u='r' c='OSError_str'/>
<use f='codebrowser/Objects/exceptions.c' l='1121' u='r' c='OSError_str'/>
